internal/locale/locales/en-us.yaml

```diff
@@ -1876,17 +1876,17 @@ flag_state_use_path_description:
 arg_state_use_namespace:
   other: org/project
 arg_state_use_namespace_description:
-  other: The org/project namespace of the project that you wish to use. If the project has already been checked out, you can simply use the project's name instead of the entire namespace.
-err_use_cannot_determine_projects_dir:
+  other: The org/project namespace of the project that you wish to use, or just the project name if previously used
```

Comment:
```suggestion
  other: The org/project namespace of the project that you wish to use, or just the project name if previously used.
```

Reply:
None of the other flag descriptions end in a period. This would stick out, no?

Reply:
Sure, go with what is best. I just scanned and thought the punctuation was missing.